components/formats-gpl/matlab/bfCheckJavaPath.m

```diff
@@ -77,9 +77,9 @@
 if status
     % Read Bio-Formats version
     if is_octave()
-        version = char(java_get('loci.formats.FormatTools', 'VERSION'));
+        version = java_get('loci.formats.FormatTools', 'VERSION');
     else
-        version = char(loci.formats.FormatTools.VERSION);
+        version = loci.formats.FormatTools.VERSION;
```

Comment:
Unfortunately on MATLAB R2014b at least, the conversion does not seem to happen by default

```
>> version = loci.formats.FormatTools.VERSION;
>> class(version)

ans =

java.lang.String
```

which is why the casting was there in the first place.